workspace_host/interface.js

```diff
@@ -764,8 +765,27 @@ async function _getInitialZipAsync(workspace) {
     debug(`Downloading s3Path=${s3Path} to zipPath=${zipPath}`);
     await _downloadFromS3Async(zipPath, s3Path);
 
+    try {
+        await fsPromises.lstat(localPath);
```

Comment:
Can we skip this `lstat()` call and just always call `mkdir()` and `_workspaceFileChangeOwnerAsync()`?